test262/index.html

```diff
@@ -109,6 +109,18 @@ <h2>test262 parser tests</h2>
           <canvas id="chart-test262-parser-tests"></canvas>
         </div>
       </section>
+      <section>
+        <h2>test262 performance</h2>
+        <div class="chart-wrapper">
+          <canvas id="chart-test262-performance-tests"></canvas>
```

Comment:
Let's drop the `-tests` suffix here, I only added that to the parser tests because that's what they call the repo (https://github.com/tc39/test262-parser-tests).

```suggestion
          <canvas id="chart-test262-performance"></canvas>
```